app/views/miq_request/_prov_field.html.haml

```diff
@@ -48,6 +48,16 @@
     .form-group
       %label.col-md-2.control-label{:valign => "top"}
         = field_hash[:description]
+        - if field_hash[:help]
+          %a{:tabindex => 0,
+             :role             => "button",
+             :"data-toggle"    => "popover",
+             :"data-trigger"   => "focus",
+             :"data-html"      => "true",
+             :title            => "",
+             :"data-content"   => "#{field_hash[:help]}",
+             :"data-placement" => "top"}
+            %span.fa.fa-info-circle
```

Comment:
Thanks @himdel, it worked inside the haml.  I kinda agreed with you that the multiple initialization was kinda ugly, so I assigned an id element of `popover-#{field_hash.object_id}' and then selected that element immediately afterwards by jquery. That should help ensure that each element is initialized exactly once.

Can you review the last commit I added? If it's fine with you I'll squash the commits together and remove the WIP tag.